hathor/websocket/factory.py

```diff
@@ -196,6 +196,8 @@ def serialize_message_data(self, event, args):
             tx = data['tx']
             data = tx.to_json_extended()
             data['is_block'] = tx.is_block
+            # XXX: needed to check if the transaction is from the mempool
```

Comment:
I didn't understand this.

Reply:
The daemon will get all transactions from the websocket, he needs some way to check if the tx he received is from the mempool or not (to start a mempool sync). This `first_block` metadata being null is enough to assert that the tx is from the mempool.

Reply:
So maybe we should remove the XXX from the comment? I think this is a normal comment and you could add the part of the wallet service daemon using this for the mempool

Reply:
I'm fine either with `XXX` or without, since this isn't surprising maybe without is better but it's subjective. I agree that just mentioning that the wallet service needs this is enough.

Reply:
Removed the `XXX`